### DISCUSSION TITLE
mpv继承播放倍速功能

### DISCUSSION THREAD

**Discussion by WuYoue** [dyphire/mpv-config#150] [Ideas]
我在使用你修改的mpv发现非常好用，尤其是长按倍速和播放记录以及UI都很好，所以抛弃了potplay。
使用下来发现没有继承上一个视频倍速的功能，有些遗憾，例如现在播放的视频设置了1.5倍速，在播放完后，下个视频自动继承上一个视频的1.5倍速。
自己动手钻探了一天，没有你的才华，所以用脚本没有实现该功能。

**Comment by dyphire** [comment]
播放下一文件时倍速会被重置是因为配置中的选项设置：
https://github.com/dyphire/mpv-config/blob/1f7bde73b1ba91d83881d8ce0f62c462da7b63f9/mpv.conf#L120-L121

从选项参数中移除`speed`值即可

**Reply by WuYoue** [reply]
好的，修改后确实可以实现该功能，非常感谢。

**Reply by WuYoue** [reply]
目前有个新的问题，在长按倍速播放以后，倍速会回到1倍速，例如当前设置1.2倍速，长按倍速播放后变成了1倍速，请问这个应该怎么解决呀？
